temporal/t.upgrade/t.upgrade.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+############################################################################
+#
+# MODULE:       t.upgrade
+# AUTHOR(S):    Martin Landa, Markus Neteler
+#
+# PURPOSE:      Upgrade of TGRASS DB
+# COPYRIGHT:    (C) 2019 by Martin Landa, and the GRASS Development Team
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#############################################################################
+
+# %module
+# % description: Upgrades the version of a space time dataset (TGRASS DB).
+# % keyword: temporal
+# % keyword: metadata
+# % keyword: extent
+# % keyword: time
+# %end
+
+import sys
+import grass.script as grass
+
+
+def main():
+    # lazy imports
+    import grass.temporal as tgis
```

Comment:
Black seems to want a blank line after import.

Reply:
thanks, fixed